app/src/main/java/com/example/apprenticeshipcrypto/overview/MainFragment.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.apprenticeshipcrypto.overview
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.example.apprenticeshipcrypto.R
+
```

Comment:
Try avoiding this kind of blank spaces in your code, it could turn code readability more difficult to understand 😄